Dockerfile.preview

```
@@ -0,0 +1,129 @@
+RUN mkdir -p /var/run/postgresql && \
+    chown -R postgres:postgres /var/run/postgresql && \
+    chmod 2775 /var/run/postgresql
+
+# Copy built artifacts
+COPY --chown=rails:rails --from=build "${BUNDLE_PATH}" "${BUNDLE_PATH}"
+COPY --chown=rails:rails --from=build /rails /rails
+
+# Create preview entrypoint script inline
+RUN cat > /rails/bin/preview-entrypoint << 'ENTRYPOINT_EOF'
+#!/bin/bash
+set -e
```

Hadolint (2.14.0)

[error] unexpected 's'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)